lib/internal/util/comparisons.js

```diff
@@ -345,6 +349,21 @@ function getEnumerables(val, keys) {
   return ArrayPrototypeFilter(keys, (key) => hasEnumerable(val, key));
 }
 
+function partialSymbolEquiv(val1, val2, keys2) {
+  const symbolKeys = getOwnSymbols(val2);
+  if (symbolKeys.length !== 0) {
+    for (const key of symbolKeys) {
```

Comment:
i would assume a for loop is faster (as well as being more robust) than a for-of loop?

Reply:
AFAIK it is not noticeable in the benchmarks, I can check that again though.